edm4eic.yaml

```diff
@@ -370,6 +370,15 @@ datatypes:
     OneToManyRelations:
       - edm4eic::TrackerHit hits            // Hits in this measurement (single or clustered)
 
+  edm4eic::TrackSeed:
+    Description: "save seed info from the realistic seed finder"
+    Author: "S. Li, B. Schmookler, J. Osborn"
+    Members:
+      - edm4hep::Vector3f  perigee   // vector to create the perigee (line surface)
+    OneToManyRelations:
+      - edm4eic::TrackerHit hits  // tracker hits that are used to form the seed 
+    OneToOneRelations:
+      - edm4eic::TrackParameters params  // initial track parameters
```

Comment:
Can you stay with the style elsewhere in this file?
- description capitalized and saying *what* is stored, not what it *does*, so no verb
- comments aligned vertically, starting with capital letter